web-animations-2/Overview.bs

```diff
@@ -969,6 +972,30 @@ Add the following sentence:
 
 </div>
 
+### Calculating the progress of an animation ### {#the-progress-of-an-animation}
+
+  The [=animation/progress=] of an [=animation=] is calculated as follows:
+
+  <dl class=switch>
+
+  : If <em>any</em> of the following are true:
+    * the animation does not have an [=animation/associated effect=], or
+    * the animation's associated effect's [=end time=] is infinite, or
+    * the animation's associated effect's [=end time=] is zero, or
+    * the animation's [=animation/current time=] time is an [=unresolved=] time value,
+  ::
+      The animation's [=animation/progress=] is null.
+  : Otherwise,
+  ::
+    <blockquote>
+      <code><a for="animation">progress</a> = [=animation/current time=] / <var>effect end time</var>
+      </code>
+    </blockquote>
+
+    Where |effect end time| is the current [=end time=] of the associated effect.
```

Comment:
Again we could replace `effect end time` with `|animation|'s [=associated effect end=]`.

Reply:
Done.